Firmware/MotorControl/motor.hpp

```diff
@@ -29,6 +29,7 @@ class Motor : public ODriveIntf::MotorIntf {
         float Id_measured; // [A]
         float I_measured_report_filter_k;
         float max_allowed_current; // [A]
+        float max_allowed_torque; // [Nm]
```

Comment:
"max allowed current" is a hardware limit, as noted earlier, probably best to keep the current limits as current and not torque.